examples/config-rust/.cargo/config.toml

```diff
@@ -0,0 +1,2 @@
+[build]
```

Comment:
Do we need another example for this? Can it be rolled into an existing one?

Reply:
Probably not, just following convention. I can roll this into `http-rust`